aten/src/ATen/native/DistributionTemplates.h

```diff
@@ -262,4 +260,29 @@ Tensor normal_impl(const Tensor& mean, const Tensor& std, Generator gen) {
   return ret;
 }
 
+// ==================================================== Uniform =======================================================
+
+template<template<typename> class uniform_kernel, typename RNG>
+at::Tensor& uniform_impl_(at::Tensor& self, double from, double to, at::Generator generator) {
+  AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, self.scalar_type(), "check_uniform_bounds", [&] {
```

Comment:
why does the complex handling not show up here? I know you made the changes here: https://github.com/pytorch/pytorch/pull/35585

Reply:
why should it be here?